apps/site/assets/css/core.scss

```diff
@@ -272,3 +273,10 @@ div {
     }
   }
 }
+
+.l-flex-container {
+  display: flex;
+}
+.l-flex-one {
+  flex: 1;
+}
```

Comment:
Let's either make these into utility classes (move to `_utilities.scss` and prefix with `u-`) _or_ move these styles to the `_events.scss` file (+ renaming the classes to use the `m-event__` convention).

Reply:
Ok great!  I think a simple, universally accessible flex classes could be useful in the future.  Is that the idea of the utility classes?

Reply:
Yep, exactly 😄